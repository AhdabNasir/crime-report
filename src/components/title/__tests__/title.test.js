import React from 'react';
import renderer from 'react-test-renderer';
import Title from '..';

describe('<Title/>', () => {
  it('renders <Title/> component', () => {
    const tree = renderer
      .create(<Title title="Page title" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
