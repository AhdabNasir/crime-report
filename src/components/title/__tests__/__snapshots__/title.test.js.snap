// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title/> renders <Title/> component 1`] = `null`;
